### ISSUE TITLE
Maven 3 requirement ?

### ISSUE THREAD

**Issue by soliad** [bytedeco/javacpp#93]
Hi,

is this possible to drop the maven 3 requirement to 3.0.3 or 3.0.0 ?
(I got some constraint on my side that require me to use the version 3.0.3)

I just build the master with 3.0.3 and I don't see any issue

Regards

**Comment by saudet** [comment]
If we lower the prerequisite to 3.0.0, the only info we get from `mvn versions:display-plugin-updates` is
```
[INFO] Require Maven 3.0.4 to use the following plugin updates:
[INFO]   org.sonatype.plugins:nexus-staging-maven-plugin ............... 1.6.6
```
So yes it's not a problem as long as you're not going to deploy. Do you need to deploy? Or what is the issue?

**Comment by soliad** [comment]
The issue is when I use the javacpp maven plugin, I got a failure just because I use maven 3.0.3 to build my wrapper.

Basically I would have to rebuild javacpp and upload it to our internal nexus to bypass the requirement.
(Will work as we don't use nexus-staging-maven-plugin 1.6.6)
But I would prefer to use directly the official version.

**Comment by saudet** [comment]
Right, I'll lower that to 3.0

**Comment by soliad** [comment]
Thanks a lot

**Comment by saudet** [comment]
Fixed in version 1.2. Thanks for reporting!
